nixos/modules/security/acme/default.nix

```diff
@@ -345,6 +345,9 @@ let
       serviceConfig = commonServiceConfig // {
         Group = data.group;
 
+        # Let's Encrypt allows 5 retries per hour, this avoids eating them all up if something is misconfigured upon the first try
+        RestartSec = 15 * 60;
```

Comment:
As there are several rate limits imposed by Let's Encrypt, it might make sense to point out that you're specifically referring to the [Failed Validation Limit](https://letsencrypt.org/docs/failed-validation-limit/) here. The important thing about that particular limit is that its scope is local and restricted to "failures per account, per hostname, per hour" – so indeed just scoped on this individual certificate renewal job of this systemd unit.

I was wondering whether the concurrency level of cert renewal needs to be factored in when calculating the restart interval, but this is __not__ necessary as stated above.

I'm fine with your approach.

Reply:
Great, thank you! I made the comment more explicit, and am landing this now :)